freqtrade/freqtradebot.py

```diff
@@ -1818,3 +1856,22 @@ def get_valid_price(self, custom_price: float, proposed_price: float) -> float:
         return max(
             min(valid_custom_price, max_custom_price_allowed),
             min_custom_price_allowed)
+
+    def discord_send(self, embeds):
+        if not 'discord' in self.config or self.config['discord']['enabled'] == False:
+            return
+        if self.config['runmode'].value in ('dry_run', 'live'):
+            webhook_url = self.config['discord']['webhook_url']
+
+            payload = {
+                "embeds": embeds
+            }
+
+            headers = {
+                "Content-Type": "application/json"
+            }
+
+            try:
+                requests.post(webhook_url, data=json.dumps(payload), headers=headers)
+            except Exception as e:
+                logger.error(f"Error sending discord message: {e}")
```

Comment:
Wrong location - please implement this properly as separate RPC module (similar to telegram).

As you seem to be using webhooks for this - i'm not sure if this is necessary at all - or if it can be handled via the (already available) webhooks - which you can fully configure via configuration (no changes necessary).